mongodbatlas/resource_mongodbatlas_federated_settings_organization_role_mapping.go

```diff
@@ -68,6 +74,32 @@ func resourceMongoDBAtlasFederatedSettingsOrganizationRoleMapping() *schema.Reso
 	}
 }
 
+func validateRoleName() schema.SchemaValidateDiagFunc {
```

Comment:
@sonlir can you also call out this new function in terraform registry docs page too to make this PR complete? https://github.com/mongodb/terraform-provider-mongodbatlas/blob/master/website/docs/r/federated_settings_org_role_mapping.html.markdown

Reply:
This page already has a full description of this feature. (roles - Specifies the Roles that are attached to the Role Mapping. Available role IDs can be found on [the User Roles Reference](https://www.mongodb.com/docs/atlas/reference/user-roles/).)
Nothing will change for end users.